package exception1;

//import java.io.IOException;

public class MyClass {

	public static void main(String[ ] args)  {
		//try {
		int a[ ] = new int[2];
		System.out.println(a[5]);
		//throw new ArrayIndexOutOfBoundsException ("Do not have Print element which is not present in the Array");
		//} catch (Exception e) {
		//System.out.println("An error occurred");
		//}
		
		}
		

}
